oaknational/Oak-Web-Application
# LESQ-119 fix(programmes): correct unit and lesson count displayed on programme…

… listing

## Description

- programmes listing page has correct unit and lesson count
- **dependant on db-tools pr merging first**

## Issue(s)

Fixes #

## How to test

1. Go to /beta/teachers-2023/key-stages/ks4/subjects/history/programmes

3. You should see unit and lesson counts are accurate

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
